client/js/widgets/label.js

```diff
@@ -27,6 +28,11 @@ export class Label extends Widget {
   applyDeltaToDOM(delta) {
     super.applyDeltaToDOM(delta);
     if(delta.text !== undefined || delta.twoRowBottomAlign !== undefined) {
```

Comment:
You need to check if delta.label !== undefined here as well to make sure it updates when just the spellcheck attribute changes.

Reply:
My apologies, I meant delta.spellCheck, sorry.

Reply:
I'm just a monkey hammering on the keyboard doing whatever the AI, Google, or you guys tell me to do.